addons/event_stand/security/ir.model.access.csv

```diff
@@ -0,0 +1,8 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_event_stand_type,event.stand.type,model_event_stand_type,,1,0,0,0
+access_event_stand_type_manager,event.stand.type.manager,model_event_stand_type,event.group_event_manager,1,1,1,1
+access_event_stand,event.stand,model_event_stand,,1,0,0,0
```

Comment:
I would go even further after @std-odoo remarks on acls

Do we need to give public users a read access on stand ? In which case if you think this is safe, some sub-fields have to gain a group. I think slot and reservation models should not be accessible (only in sudo in some compute / data preparation) to public / portal users to avoid leaking information ... in that case some fields (o2m) should have a group, and some other computed (count, availability) probably computed using a sudo (to check a bit more in details about ACLs and what we want to disclose / what is accessed through controllers and sudo).